tools/snippet_shell.py

```diff
@@ -34,13 +34,30 @@
 from mobly.controllers.android_device_lib import jsonrpc_shell_base
 
 
+def _patch_print_rpc(snippet, name):
```

Comment:
i was thinking simply defining the `help` method in the snippet client class...
would be less hacky?

Reply:
Doesn't that potentially cause issues if someone actually wants the help() response?
I could add a default argument to disable printing in that case I guess.

Reply:
Changed.

This is a bit more dangerous to do, but it's probably okay.